modules/auxiliary/gather/vbulletin_getindexablecontent_sqli.rb

```diff
@@ -0,0 +1,286 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Auxiliary
+  include Msf::Auxiliary::Report
+  include Msf::Exploit::Remote::HttpClient
+
+  HttpFingerprint = { method: 'GET', uri: '/', pattern: [/vBulletin.version = '5.+'/] }
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'vBulletin /ajax/api/content_infraction/getIndexableContent nodeid Parameter SQL Injection',
+        'Description' => %q{
+          This module exploits a SQL injection vulnerability found in vBulletin 5.x.x to dump the user
+          table information or to dump all of the vBulletin tables (based on the selected options). This
+          module has been tested successfully on VBulletin Version 5.6.1 on Ubuntu Linux.
+        },
+        'License' => MSF_LICENSE,
+        'Author' => [
+          'Charles Fol <folcharles[at]gmail.com>', # (@cfreal_) CVE
+          'Zenofex <zenofex[at]exploitee.rs>', # (@zenofex) PoC and Metasploit module
+        ],
+        'References' => [
+          ['CVE', '2020-12720'],
+        ],
+        'Platform' => 'php',
+        'Arch' => ARCH_PHP,
+        'Targets' => [
```

Comment:
Auxiliary modules do not have targets, only actions, which haven't been set:

```
msf5 auxiliary(gather/vbulletin_getindexablecontent_sqli) > show targets
[-] No exploit module selected.
msf5 auxiliary(gather/vbulletin_getindexablecontent_sqli) > show actions

Auxiliary actions:

   Name  Description
   ----  -----------


msf5 auxiliary(gather/vbulletin_getindexablecontent_sqli) >
```

Reply:
Added a DumpAll action and set the default action to DumpUser. Then removed the DumpAll module boolean option, fixed the docs and added it into the scenario. Thanks